aws/files/autoeips.py

```diff
@@ -74,10 +120,21 @@ def associate_eip(self, eips, retries=3):
 
                 success = eip.associate(
                     instance_id=self.instance_id, allow_reassociation=False)
+                self.update_standby_mode(False)
                 return success
+        logger.warning("Failed to associate with any EIP's")
+        self.update_standby_mode(True)
         return False
 
     def get_unassociated_eips(self):
+        """
```

Comment:
Just realised you already have some code to do that:

if (self.filter_addresses < 1):

The pythonic way is:

if not self.filter_addresses:

Or you could do it your way if you get the length of self.filter_addresses, e.g. len(self.filter_addresses)